colin-api/src/colin_api/models/business.py

```diff
@@ -118,6 +119,18 @@ def as_dict(self) -> Dict:
             }
         }
 
+    def as_dict_nr(self) -> Dict:
```

Comment:
please rename `as_dict_nr` to `as_dict`

Reply:
There is already `as_dict` https://github.com/bcgov/lear/blob/05d00862f9b482ac36982b99c0fe08464964271d/colin-api/src/colin_api/models/business.py#L102

Reply:
Try updating `as_dict` to follow an approach similar to legal api.  i.e. introduce an optional `slim` parameter to `as_dict`

Reply:
I updated